payments/payment-methods/adyen/src/main/java/cm/aptoide/pt/payment_method/adyen/presentation/AdyenCreditCardViewModel.kt

```diff
@@ -73,6 +77,7 @@ class AdyenCreditCardViewModel(
   private val cardConfiguration: CardConfiguration,
   private val redirectConfiguration: RedirectConfiguration,
   private val threeDS2Configuration: Adyen3DS2Configuration,
+  private val lastPaymentStateRepository: LastPaymentStateRepository,
```

Comment:
We should not use repositories from other modules.
Let's stick with our agreements on the modules architecture:
At most view models and business objects (or use cases).